component-runtime-testing/component-runtime-http-junit/src/test/java/org/talend/sdk/component/junit/http/junit4/CaptureJUnit4HttpApiTest.java

```diff
@@ -106,7 +106,6 @@ public void evaluate() throws Throwable {
                                             new InetSocketAddress("localhost", API.getPort()))));
                     connection.setConnectTimeout(30000);
                     connection.setReadTimeout(20000);
-                    connection.setRequestProperty("Accept", "*/*");
```

Comment:
Should it be removed ? Was it related to java17 ?

Reply:
changes in gttp client AFAIR